debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresOffsetContext.java

```diff
@@ -207,15 +207,18 @@ public PostgresOffsetContext load(Map<String, ?> offset) {
             final String msgType = (String) offset.getOrDefault(SourceInfo.MSG_TYPE_KEY, null);
             final Operation messageType = msgType == null ? null : Operation.valueOf(msgType);
             final Instant useconds = Conversions.toInstantFromMicros((Long) ((Map<String, Object>) offset).getOrDefault(SourceInfo.TIMESTAMP_USEC_KEY, 0L));
-            final boolean snapshot = (boolean) ((Map<String, Object>) offset).getOrDefault(SourceInfo.SNAPSHOT_KEY, Boolean.FALSE);
+            final SnapshotType snapshot = loadSnapshot((Map<String, Object>) offset);
+            boolean snapshotCompleted = Boolean.TRUE.equals(offset.get(CommonOffsetContext.SNAPSHOT_COMPLETED_KEY));
```

Comment:
How this will work in case of connector update when this field is not provided? Might be an issue for the other connectors too.

Reply:
Well, before this PR the all connectors used the `snapshotCompleted` flag so they should have the flag in their offset and during the update it will correctly managed. 

Only PostgreSQL haven't the flag before and so it will be put as `false`.